src/bots/depositor.py

```diff
@@ -75,9 +75,13 @@ def __init__(self, w3: Web3):
     def execute(self, block: BlockData) -> bool:
         self._check_balance()
 
-        module_id = get_preferred_to_deposit_module(self.w3, variables.DEPOSIT_MODULES_WHITELIST)
+        modules_id = get_preferred_to_deposit_modules(self.w3, variables.DEPOSIT_MODULES_WHITELIST)
 
-        if module_id:
+        if not modules_id:
+            # Read messages in case if no depositable modules for metrics
+            self.message_storage._receive_messages()
```

Comment:
Does that mean that if none modules are whitelisted we are not reading anything from the queue? And in case of RMQ it will be overfilled?

Reply:
Currently, yes.
This change fixes this.